style-guide.md

```diff
@@ -0,0 +1,189 @@
+# Cincytech slack community Job Description Style Guide
+
+Finding the right person for a job is difficult. And writing a job descriptions
+to attract that person is hard. Provide too much detail and no one will ever
+read it. Provide too little detail and no one would dare apply. It's an art
+form, and attempting to strike a balance between too little and too much can be
+time consuming and the results can be unpredictable.
+
+
+This document is an attempt to summarize what we, the cincytech slack community
+feel is a good job description for jobs in software development. It is not meant
+to be exhaustive or authoritative, instead it is merely a set of principles to
+keep in mind when you are writing the job description to find the perfect person you need
+for the job.
```

Comment:
After reading through this a couple times, I realized one of my issues with the tone of this document and this last sentence nails it: _I want a better job descriptions to help *individuals* identify the best fit opportunity._

Sure, the "demand" side of the transaction is important, but I'd like to focus on the "supply" side—the job/role/gig seeking *individual*.

There is certainly a bias in my thinking as I've been _independent_ for over ten years now, but I had 7 employers in my first 10 years out of college. Similarly, I've had 14 clients in my first 10 years of consulting. Given the very narrow demographic of the Slack community, I'd like to see _employee-focused_ descriptions and hope that those attributes that help "sell" a position also result in a better "buy" for the employer.